bookwyrm/models/user.py

```diff
@@ -509,6 +512,15 @@ def raise_not_editable(self, viewer):
             return
         raise PermissionDenied()
 
+    def refresh_user_sessions(self):
+        """Check sessions still exist
+        We delete them on logout but not when sessions expire"""
+
+        for sess in self.sessions.all():
+            cache_session = SessionStore()
```

Comment:
Unless I'm missing something, this object creation can safely be moved right before the loop?